share/locale/en.json

```diff
@@ -1,165 +1,165 @@
-    "gui_tor_connection_canceled": "OnionShare could not connect to Tor.\n\nMake sure you're connected to the Internet, then re-open OnionShare to set up the Tor connection.",
+    "gui_tor_connection_error_settings": "Try changing how OnionShare connects to the Tor network in the settings.",
+    "gui_tor_connection_canceled": "Could not connect to Tor.\n\nEnsure you are connected to the Internet, then re-open OnionShare and set up its connection to Tor.",
     "gui_tor_connection_lost": "Disconnected from Tor.",
-    "gui_server_started_after_timeout": "The server started after your chosen auto-timeout.\nPlease start a new share.",
-    "gui_server_timeout_expired": "The chosen timeout has already expired.\nPlease update the timeout and then you may start sharing.",
-    "share_via_onionshare": "Share via OnionShare",
+    "gui_server_started_after_timeout": "The expiry timer ran out before the server started.\nPlease start a new share.",
+    "gui_server_timeout_expired": "The expiry timer ran out.\nPlease restart it to start sharing.",
+    "share_via_onionshare": "OnionShare it",
     "gui_save_private_key_checkbox": "Use a persistent address\n(unchecking will delete any saved addresses)",
-    "gui_url_description": "<b>Anyone</b> with this link can <b>download</b> your files using the <b>Tor Browser</b>: <img src='{}' />",
-    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every share will have the same address (to use one-time addresses, disable persistence in Settings)",
-    "gui_url_label_stay_open": "This share will not expire automatically unless a timer is set.",
-    "gui_url_label_onetime": "This share will expire after the first download",
-    "gui_url_label_onetime_and_persistent": "This share will expire after the first download<br><br>Every share will have the same address (to use one-time addresses, disable persistence in the Settings)",
-    "gui_status_indicator_stopped": "Ready to Share",
-    "gui_status_indicator_working": "Starting…",
-    "gui_status_indicator_started": "Sharing",
+    "gui_url_description": "<b>Anyone</b> with this OnionShare address your files using the <b>Tor Browser</b>: <img src='{}' />",
+    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every subsequent share will be available at the same address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_url_label_stay_open": "Add a timer to automatically expire the share.",
+    "gui_url_label_onetime": "This OnionShare expires after one OnionSharing",
+    "gui_url_label_onetime_and_persistent": "This OnionShare expires after completion<br><br>Every subsequent OnionShare will be available at the same OnionShare address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_status_indicator_stopped": "Ready to OnionShare",
+    "gui_status_indicator_working": "Hyping…",
```

Comment:
lol :+1:

Reply:
'Hyping' confused me actually ?

Reply:
It was a joke, and it is quite pointless to even have anything. Every other mature program has just an animation that signifies waiting. Try opening https://github.com/micahflee/onionshare/pull/751/files  There is a spinning circle sporting the GitHub logo within it.